articles/concepts/circuits.md

```diff
@@ -109,7 +109,7 @@ Similarly, the subcircuit
 
 <!--- ![](.\media\8.svg) --->
 <!-- Can't find a way to easily center this... probably an extension needed:  -->
-![](~/media/concepts_8.png)
+![Circuit diagram representing a controlled operation](~/media/concepts_8.png)
```

Comment:
I think this is more "Circuit diagram of a classically controlled gate"

Reply:
@cgranade seems to prefer 'operation' over 'gate' for some of these descriptions, although @KittyYeungQ says they are both correct. I'm not up to speed enough yet to know how interchangeable they are.

Reply:
We've been trying to be more consistent about the use of "operation", as that refers specifically to the Q# concept. A "gate" could be a unitary matrix, a physical procedure that can be simulated by a unitary matrix, or a quantum program that instructs a device to undertake that procedure. My suggestion was thus intended to avoid equating operations with matrices used to simulate those operations, and to reduce ambiguity overall.

Reply:
I was actually referring to the classical part not the operation/gate part. Isn't the point of this diagram that it represents the controlled functionality of a classical bit (as compared to a Qubit)?

Reply:
"Circuit diagram representing a classically controlled operation."

Reply:
That sounds good to me, but I'll let @cgranade have final say :-)

Reply:
I think that works great, sorry for the delay!